testsupport/wait/host.go

```diff
@@ -162,138 +162,200 @@ func (a *HostAwaitility) UpdateUserSignupSpec(userSignupName string, modifyUserS
 }
 
 // MasterUserRecordWaitCriterion checks if a MasterUserRecord meets the given condition
```

Comment:
the comment probably needs to be changed since MasterUserRecordWaitCriterion is not a function now?

Reply:
good point! Addressed in e18eb4b for all similar structs (among other changes to improve readability of code)